platform/ui/src/components/Viewport/Viewport.jsx

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { ViewportActionBar, Notification, Button } from '@ohif/ui';
 
-const Viewport = ({ viewportIndex, onSeriesChange, studyData, children }) => {
+const Viewport = ({
```

Comment:
I think this is an unused component -- or only used for the docz "view" to replicate the UI we're targeting